nemo/collections/vlm/recipes/mllama_11b.py

```diff
@@ -134,15 +136,16 @@ def finetune_recipe(
     elif peft_scheme.lower() == 'lora':
         recipe.peft = run.Config(
             vlm.LoRA,
-            freeze_vision_model=False,
+            freeze_vision_model=True,
             target_modules=[
-                "*.language_model.*.linear_qkv",
-                "*.language_model.*.linear_q",
-                "*.language_model.*.linear_kv",
-                "*.language_model.*.linear_proj",
-                "*.language_model.*.linear_fc1",
-                "*.language_model.*.linear_fc2",
+                "linear_qkv",
+                "linear_q",
+                "linear_kv",
             ],
+            dim=8,
+            alpha=32,
+            dropout=0.05,
+            dropout_position="pre",
```

Comment:
I think it would be good to link the source of these values from meta here

Reply:
they don't have this in checkpoint. They have a separate repo/script for this info